.ado/templates/apple-xcode-select.yml

```diff
@@ -1,8 +1,12 @@
 #
-# Task Group: XCode select proper version
+# Task Group: Xcode select proper version
 #
+parameters:
+  xcode_path_override: '' # Leaving this blank results in the repo default being applied
+
 steps:
-  - bash: |
-      sudo xcode-select --switch '/Applications/Xcode_11.4.1.app'
-    displayName: 'Switch to XCode 11.4.1'
+  - bash: scripts/xcode_select_current_version.sh
```

Comment:
Where is this script?

Reply:
...derp. Forgot to bring that over. I was copying what we did in FluentUI Apple when I noticed they don't explicitly set an xcode version and use the default. I'll bring it over